src/boss/cloudvm/local_worker.go

```diff
@@ -6,18 +6,22 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 
 	"github.com/open-lambda/open-lambda/ol/common"
 )
 
 // WORKER IMPLEMENTATION: LocalWorker
 type LocalWorkerPoolPlatform struct {
```

Comment:
I think we need to write thread-safe code because we're supposed to support concurrent creation of instances: https://github.com/open-lambda/open-lambda/blob/main/src/boss/cloudvm/worker_pool.go#L122.  So LocalWorkerPoolPlatform should probably have a lock to protect nextWorkerPort and other state that might change.